### ISSUE TITLE
TS31: Error message regarding correct email syntax doesn't include the syntax

### ISSUE THREAD

**Issue by HannahCastro** [vasavah/vsv-fli-pass-uat#126]
**Describe the bug**
The error message regarding encoding an email address using the email syntax didn't contain the email syntax format.

**Expected behavior**
Error message should contain the email syntax format. Example is "example@example.com".

**Screenshots**
![image](https://user-images.githubusercontent.com/51902789/80957448-b76ce780-8e35-11ea-889b-6a5d2447bae2.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Google Chrome
 - Version 81.0.4044.129

**Comment by VCI-Jojo** [comment]
@HannahCastro This is related to tickets#106 and #111 for V1.1

**Comment by HannahCastro** [comment]
@VCI-Jojo Previously (UAT Server testing), error prompt contained the email syntax format but was removed in Prod Server testing. Will this really be for V1.1 now?

**Comment by VCI-Jojo** [comment]
> @VCI-Jojo Previously (UAT Server testing), error prompt contained the email syntax format but was removed in Prod Server testing. Will this really be for V1.1 now?

@HannahCastro If it will only be the wordings for the "error prompt" then we can include in V1.0, so kindly confirm. Thank you.

**Comment by HannahCastro** [comment]
@VCI-Jojo Yes, this is for the wording. Previously, system checked if user didn't include the domain name. System will show prompt stating "You must enter an email address in this format: _example@example.com_". Italicized words were removed in Prod Server.

**Comment by HannahCastro** [comment]
System required the user to follow the email forward in the email invitation. This is now for closing.
![image](https://user-images.githubusercontent.com/51902789/115805626-eb129a00-a417-11eb-8b0a-153181ec46f4.png)

For reference:
![image](https://user-images.githubusercontent.com/51902789/111432436-fd722780-8737-11eb-83e0-97a9e2b57afc.png)
